ct-app/postman/postman_tasks.py

```diff
@@ -39,8 +81,17 @@ async def async_send_1_hop_message(
     :param node_index: Index of the node in the list of nodes.
     """
 
+    # at the first iteration, the timestamp is set to the current time. At each task
+    # transfer, the method will check if the timestamp is recent enough to consider
+    # trying to send a message
+    if time.time() - timestamp > 60 * 60 * 2:  # timestamp is older than 2 hours
```

Comment:
Add comment about `timestamp` parameter to the function.

Reply:
Done !